src/request/remove-key/index.html

```diff
@@ -22,6 +22,7 @@
     <script defer bundle-common src="../../lib/KeyInfo.js"></script>
     <script defer bundle-common src="../../lib/Errors.js"></script>
     <script defer bundle-common src="../../lib/ErrorConstants.js"></script>
+    <script defer bundle-common src="../../lib/SignMessageConstants.js"></script>
```

Comment:
why is this needed for this request, and the others. I only comment here, but question is for all requests except for `sign-message`.

Reply:
ah nevermind. Key.js needs it so yes, it is needed.